app/celery/tasks.py

```diff
@@ -400,7 +424,7 @@ def save_sms(self, service_id, notification_id, encrypted_notification, sender_i
 
 @notify_celery.task(bind=True, name="save-emails", max_retries=5, default_retry_delay=300)
 @statsd(namespace="tasks")
-def save_emails(self, service_id: str, encrypted_notifications: List[Any]):
+def save_emails(self, service_id: str, encrypted_notifications: List[Any], receipt: str):
```

Comment:
It seems the receipt param could be `None`, right? Could we add a default value to it?

Reply:
👍  `Optional[str]` as above